test/test_helper.rb

```diff
@@ -87,4 +87,61 @@ def teardown
   def create_user
     User.create(username: "ryan", password: "waffles", address: "1111 Maple Rd", city: "Denver", state: "CO", zip: 82020)
   end
+
+  def add_2
+    visit items_path
+    item1 = Item.find_by(title: "pour over0")
+    item2 = Item.find_by(title: "drip0")
+
+    within "#item#{item1.id}" do
+      click_button "Add to Cart"
+    end
+
+    within "#item#{item2.id}" do
+      click_button "Add to Cart"
+    end
+
+    click_button "View Cart"
+    click_button "Checkout"
+    fill_in 'Username', with: 'ryan'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.